java/src/org/openqa/selenium/firefox/GeckoDriverService.java

```diff
@@ -219,15 +221,15 @@ protected void loadSystemProperties() {
     @Override
     protected List<String> createArgs() {
       List<String> args = new ArrayList<>();
-      args.add(String.format("--port=%d", getPort()));
+      args.add(new Formatter(Locale.US).format("--port=%d", getPort()).toString());
```

Comment:
The issue is we've to flush the formatter everytime because when I triumph D using one formatter object without flushing it
It causes the strings to append

Reply:
@pujagani  I refactored the code to use String.format again with Locale.US